apertium-crh.crh.lexc

```diff
@@ -152,7 +180,6 @@ Multichar_Symbols
 %{b%}
 %{p%}
 
-%{☭%}
```

Comment:
Why was this removed?

Reply:
Sorry added it back must have been deleted while editing.